src/python/tests/core/bot/fuzzers/libFuzzer/libfuzzer_run_test.py

```diff
@@ -61,6 +61,11 @@ class FuzzerTest(builtin_test.BaseEngineFuzzerTest):
   def test_run(self):
     """Test running libFuzzer fuzzer."""
     libfuzzer = fuzzer.LibFuzzer()
+
+    # Don't test mutator_plugins here.
```

Comment:
I test it elsewhere so it isn't needed here. But when the test ran before it would spend a really long time and then error out when talking to GCS. Maybe we should get this working?